### PR TITLE
Transitioned to USDOT fork of asn1c repository

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
USDOT has recently created a fork of the mouse07410/asn1c project. To maintain internal references, the project needs to switch to using this new fork.

## Solution
The project now uses https://github.com/usdot-fhwa-stol/usdot-asn1c as its ASN.1 Compiler. References to the submodule have been updated in the relevant scripts and Dockerfiles.

## Testing
- Installing the asn1c tool has been verified to work
- Generating the J2735 implementation/header files has been verified to work for 2016 & 2020
- The docker image can be built successfully using the new 2020 generated files
- Unit tests are verified to be passing.

## Note
It should be noted that using the latest commit in usdot-fhwa-stol/usdot-asn1c results in failure to compile the ACM. To work around this, the following commit is being referenced:
https://github.com/usdot-fhwa-stol/usdot-asn1c/commit/e74487cd9b585d19f2c15ece949f754e3733cb59

### CODE REVIEW

File: src/tests.cpp [hunk 1]
```diff
@@ -38,7 +38,7 @@ const char *ONE609_BSM_HEX = "038081B1001480AD562FA8400039E8E717090F9665FE1BACC3
 const char *ASD_ONE609_HEX = "44400000000084782786283B90A7148D2B0A89C49F8A85A7763BF8423C13C2107E1C0C6F7E2C0C6F16A070103620029015AAC5F50800073D1CE2E121F2CCBFC375986FFFFFFFFE0007775FBF43F4200FFFF000000000004042983820082FFFFFFFD049C20147FFFFFFFD128420408FFFFFFFD2454204D480287FFD2BAC2084680BAFFFD4EAC2064580B33FFD5BF42072380D6BFFD6FCC2079681407FFDA424206778115BFFDB34C205D0811CBFFDBC5C2057B8110BFFDBE142001780337FFEFE643FFFF800BBFFF8AB43FFFFFFFFFFFFBA3420080FFFFFFFFC345FFFC00000";
 
 /**
- * After switching to using `mouse07410/asn1c` (a fork of `vlm/asn1c`), this unit test is failing.
+ * After switching to using `mouse07410/asn1c` (a fork of `vlm/asn1c`), this unit test is failing. Note: We are now using the `usdot-fhwa-stol/usdot-asn1c` fork.
```
Comment (drewjj) [inline src/tests.cpp:41]: Does it make sense to continue to reference the mouse documentation going forward if we are switching?

Comment (dmccoystephenson) [reply src/tests.cpp]: These comments have been updated and now only `usdot-fhwa-stol/usdot-asn1c` is mentioned.

**Review by drewjj — APPROVED** [review]
Nice, looking good now.

**Review by mwodahl — APPROVED** [review]
This looks good to me!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**